v1/ao/internal/reporter/reporter_grpc.go

```diff
@@ -707,10 +742,21 @@ func (r *grpcReporter) getSettings(ready chan bool) {
 		r.metricConnection.resetPing()
 
 		if err != nil {
-			OboeLog(INFO, fmt.Sprintf("Error calling GetSettings(): %v", err))
-			// some server connection error, attempt reconnect
-			r.reconnect(r.metricConnection, GETSETTINGS)
+			// gRPC handles the reconnection automatically.
+			failsNum++
+			if failsNum > grpcRetryLogThreshold && !failsPrinted {
+				OboeLog(WARNING, fmt.Sprintf("Error calling PostEvents(): %v", err))
```

Comment:
Should it be something like `Error calling getSettings()` in this and below log messages?

Reply:
Yes. Corrected.

Reply:
👍